### DISCUSSION TITLE
Is there a reason NOT to use the "search" HTML element for the search partial?

### DISCUSSION THREAD

**Discussion by wolfspyre** [jpanther/congo#698] [Q&A]
Using the 'header' element at `layouts/partials/search.html:10` and`layouts/partials/search.html:29` causes a minor whinge in safari's accessibility tests as `header` is declared twice. 

Is there a reason **NOT** to use the `search` element[^1] instead of the header element?
I looked thru the list of html entity types[^2]; an this felt like a decent fit; but wth do I know? :)

I don't see any problems after changing it from header to search while using mobile/desktop safari on osx/ios/ipad. .

(admittedly I didn't test on ffox, edge, or any linux/win hosts, but.....)

[^1]: <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/search>
[^2]: <https://developer.mozilla.org/en-US/docs/Web/HTML/Element>

**Comment by jpanther** [comment]
I didn't even know the `search` element existed! 😆
